collector/agentPoller.go

```diff
@@ -115,39 +114,39 @@ func NewAgent(ipCommand string, port int, pollPeriod int, topic string) (Agent,
 
 // Run runs an Agent Poller which periodically produces data retrieved from a local Mesos Agent.
 // This function should be run as a gofunc.
-func (a *Agent) Run(recordsChan chan<- *AvroDatum, stats chan<- statsd.StatsEvent) {
+func (a *Agent) Run(recordsChan chan<- *AvroDatum) {
```

Comment:
This isn't a MVP requirement, but it would be great to go straight to something like a `Datum{}` struct or `JSONDatum{}` struct instead of Avro. But I understand if that's not possible right now due to the Great Refactor (TM).